### PR TITLE
Patient Record & ANC Subs Translations

### PR DESCRIPTION
**Pull request by Diama1**
#385
#384

### CODE REVIEW

**Comment by Diama1** [comment]
Screenshot from 2022-08-26 09-23-54

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -3668,17 +3668,17 @@ translationSet trans =
...
         DiagnosedAtAnotherFacilityPrefix ->
             { english = "You were diagnosed with"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Wasuzumwe"
             }
 
         DiagnosedAtAnotherFacilitySuffix ->
             { english = "at another facility and were given medication. Which medication was given?"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Ku rindi vuriro wagiyeho ugahabwa imiti. Ni iyihe miti wahawe"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: Seems to me we need `?` at the end

File: client/src/elm/Translate.elm [hunk 2]
```diff
@@ -6270,27 +6270,27 @@ translationSet trans =
...
         MedicationDistributionHelperTrichomonasOrBacterialVaginosis ->
             { english = "This patient has signs of possible Trichomonas or Bacterial Vaginosis"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Umubyeyii afite ibimenyetso bishobora kuba ari ibya Tirikomonasi cyangwa Mikorobe zo mu nda ibyara"
             }
 
         MedicationDistributionNoticeGonorrhea ->
             { english = "Note: It is also recommended to prescribe the partner"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Icyitonderwa:Ni ngombwa kuvura uwo babana/bashakanye"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: Add space after `:`

File: client/src/elm/Translate.elm [hunk 3]
```diff
@@ -11682,7 +11682,7 @@ translationSet trans =
 
                 TreatmentHealthEducationForHeartburn ->
                     { english = "Not dispensing medicine. Follow health education protocols."
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Witanga umuti. Kurikiza amabwiriza ajyanye n'inyigisho z'buzima"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: Should end with `.`

File: client/src/elm/Translate.elm [hunk 4]
```diff
@@ -11628,14 +11628,14 @@ translationSet trans =
                     -- Names of Medication, therefore,
                     -- no translation is needed.
                     { english = "Erythromycin (500mg)"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Erythromicine"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: This is wrong

File: client/src/elm/Translate.elm [hunk 5]
```diff
@@ -11628,14 +11628,14 @@ translationSet trans =
...
                     }
 
                 TreatmentAzithromycin ->
                     -- Names of Medication, therefore,
                     -- no translation is needed.
                     { english = "Azithromycin (2g)"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Azithromycine"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: This is wrong

File: client/src/elm/Translate.elm [hunk 6]
```diff
@@ -11518,7 +11518,7 @@ translationSet trans =
 
                 TreatmentErythromycin ->
                     { english = "by mouth 4x a day for 14 days"
-                    , kinyarwanda = Just "mu kanwa inshuro enye ku munsi mu minsi 14"
+                    , kinyarwanda = Just "Mu kanwa inshuro enye ku munsi mu minsi 14"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: I think that whatever starts with small letter in English should start with small letter as well.
If I'm right, change, if not, ignore.

File: client/src/elm/Translate.elm [hunk 7]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
 
                 TreatmentAluminiumHydroxide ->
                     { english = "1 tablet by mouth 3x a day for 7 days"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Kunywa ikinini 1 inshuro ku munsi mu minsi 7"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 8]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
...
                     }
 
                 TreatmentNitrofurantoin ->
                     { english = "by mouth 2x a day for 7 days"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu kanwa inshuro 2 ku munsi mu minsi 7"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 9]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
...
                     }
 
                 TreatmentAmoxicillin ->
                     { english = "by mouth 3x a day for 7 days"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu kanwa inshuro 3 ku munsi mu minsi 7"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 10]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
...
                     }
 
                 TreatmentClotrimaxazole200 ->
                     { english = "vaginally every night x 3 night"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu gitsina buri joro mu majoro 3"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 11]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
...
                     }
 
                 TreatmentClotrimaxazole500 ->
                     { english = "vaginally one time"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Inshuro imwe mu gitsina"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 12]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
...
                     }
 
                 TreatmentMethyldopa2 ->
                     { english = "by mouth 2x a day"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu kanwa inshuro 2 ku munsi"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 13]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
...
                     }
 
                 TreatmentMethyldopa3 ->
                     { english = "by mouth 3x a day"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu kanwa inshuro 3 ku munsi"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 14]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
...
                     }
 
                 TreatmentMethyldopa4 ->
                     { english = "by mouth 4x a day"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu kanwa inshuro 4 ku munsi"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 15]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
...
                     }
 
                 TreatmentHypertensionAddCarvedilol ->
                     { english = "by mouth 2x a day"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu kanwa inshuro 2 ku munsi"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 16]
```diff
@@ -11533,52 +11533,52 @@ translationSet trans =
...
                     }
 
                 TreatmentHypertensionAddAmlodipine ->
                     { english = "by mouth 1x a day"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu kanwa inshuro 1 ku munsi"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: start with small letter

File: client/src/elm/Translate.elm [hunk 17]
```diff
@@ -10719,32 +10719,32 @@ translationSet trans =
...
                 OutsideCareMedicationErythromycin ->
                     { english = "Erythromycin (500mg)"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Erythromicine"
                     }
 
                 OutsideCareMedicationAzithromycin ->
                     { english = "Azithromycin (2g)"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Azithromycine"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: This is wrong

File: client/src/elm/Translate.elm [hunk 18]
```diff
@@ -10719,32 +10719,32 @@ translationSet trans =
...
                 OutsideCareMedicationPenecilin3 ->
                     { english = "Penicillin (2.4 million units)"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Penisilini (Miliyoni 2.4)"
                     }
 
                 OutsideCareMedicationErythromycin ->
                     { english = "Erythromycin (500mg)"
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Erythromicine"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: This is wrong

File: client/src/elm/Translate.elm [hunk 19]
```diff
@@ -13160,12 +13160,12 @@ translationSet trans =
...
                 TreatmentQuinineSulphate ->
                     { english = "At the previous visit you were given Quinine Sulphate per os 10 mg/kg/dose, 3 times a day for 7 days."
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu isura rishize wahawe umuti wa Kinini 10mg ku kilo, gatatu ku munsi mu minsi irindwi."
                     }
 
                 TreatmentCoartem ->
                     { english = "At the previous visit you were given Coartem, 4 tablets by mouth twice per day x 3 days."
-                    , kinyarwanda = Nothing
+                    , kinyarwanda = Just "Mu isura rishize wahawe AL (Kowaritemu), ibibini bine (4) byo kunywa mu kanwa inshuri ebyiri ku munsi mu minsi itatu"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: should end with `.`

**Review by anvmn — CHANGES_REQUESTED** [review]

**Review by anvmn — APPROVED** [review]

**Pull request merged**